front/src/shared/Catalogue.js

```diff
@@ -0,0 +1,28 @@
+import ImageSlider from "./ImageSlider/ImageSlider";
+import { productsLinks } from "../constants/imageLinks";
+import Card from "./Card";
+
+export default function Catalogue() {
+  const links = Object.entries(productsLinks);
+  return (
+    <ImageSlider className="gap-2">
+      {links.map(([key, url], index) => {
+        return (
+          <Card
+            key={key}
+            url={url}
+            index={index}
+            altText="Product card"
+            title="Laptop Lenovo G470"
+            originalPrice={100}
+            offerPrice={59.9}
+            specialPrice={39.9}
+            discount={40}
```

Comment:
👋 Let's not forget to pass this hardcoded values to a constant. Later these should come from the backend.

Reply:
LGTM! Ok Thanks for observation, ready